### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by FRSNVIDAL** [Fictiverse/Redream#15] [General]
aid! Does anyone have a tutorial on how to use it?

**Comment by MackinationsAi** [comment]
Hey FRSNVIDAL,

Here is a pretty good demo/tutorial video on how to use Fictiverse w/ Automatic1111 - https://www.youtube.com/watch?v=AB7MjRKEosw

Make sure you have these Command Line Arguments/Settings added to your webui-user.bat file.

set COMMANDLINE_ARGS=--xformers --opt-split-attention --no-half-vae --api --allow-code

Here is a link to all the possible cmd optimization args you can add - https://github.com/AUTOMATIC1111/stable-diffusion-webui/wiki/Command-Line-Arguments-and-Settings

Once you've saved the .bat file close it, launch Automatic1111 & wait till the webui is fully loaded. Next (if you don't already have it installed you need to go to extensions & install ControlNet - https://github.com/Mikubill/sd-webui-controlnet.git - then restart Automatic1111). 

Next, if you haven't done so already you need to install .NET 6.0 Framework - https://dotnet.microsoft.com/en-us/download/dotnet/6.0

After you've done all of that, you need to make sure you have the right Fictiverse Redream release - https://github.com/Fictiverse/Redream/releases/tag/1.12 - download the asset file named Fictiverse_Redream_1.12.7z or just load the following link - https://github.com/Fictiverse/Redream/releases/download/1.12/Fictiverse_Redream_1.12.7z

Unzip the file & place it anywhere really (though I would suggest putting it in the same drive/folder you have Automatic1111 in keeps things a bit more organized). Then open up the Fictiverse_Redream_1.12 folder & open Redream.exe (before you open up Redream make sure that Automatic is up & running first).

From there it is pretty straight forward - https://github.com/Fictiverse/Redream#how-to-use

A few tips, make sure you have a decent positive & negative prompt added in at the top or use the [T] button at the top right to do an integrated clip of last image you generated, make sure the save frame button near the top right is **ON & GREEN** or nothing you generate will be saved in the Frames folder (it's the first thing I do after opening), the bottom right hamburger button is settings & that's where you will find all of the models that you have installed for stable diffusion & you can pick which one you want to use, play around with everything till you start getting good generations!

Hope all this helps! _M
